cpp/libcudf_kafka/cmake/thirdparty/CUDF_KAFKA_GetCUDF.cmake

```diff
@@ -14,21 +14,39 @@
 # limitations under the License.
 #=============================================================================
 
+function(cudfkafka_save_if_enabled var)
+    if(CUDF_KAFKA_${var})
+        unset(${var} PARENT_SCOPE)
+        unset(${var} CACHE)
+    endif()
+endfunction()
+
+function(cudfkafka_restore_if_enabled var)
+    if(CUDF_KAFKA_${var})
+        set(${var} ON CACHE INTERNAL "" FORCE)
+    endif()
+endfunction()
+
 function(find_and_configure_cudf VERSION)
+    cudfkafka_save_if_enabled(BUILD_TESTS)
+    cudfkafka_save_if_enabled(BUILD_BENCHMARKS)
     CPMFindPackage(NAME cudf
         VERSION         ${VERSION}
-        GIT_REPOSITORY  https://github.com/rapidsai/cudf.git
-        GIT_TAG         branch-${VERSION}
+        GIT_REPOSITORY  https://github.com/trxcllnt/cudf.git
+        # GIT_TAG         branch-${VERSION}
+        GIT_TAG         fix/cmake-always-export-cudftestutil
```

Comment:
This needs to be reverted before merging 😄

Reply:
Yes definitely, this was just for testing